### PR TITLE
[MBL-16314][Teacher] Publish/Unpublish assignments

### PR DESCRIPTION
**Pull request by kristofnemere**
refs: MBL-16314
affects: Teacher
release note: none

Test plan: In the ticket

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [x] A11y checked
- [x] Approve from product or not needed

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — CHANGES_REQUESTED** [review]

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/EditAssignmentDetailsFragment.kt [hunk 1]
```diff
@@ -187,14 +187,10 @@ class EditAssignmentDetailsFragment : BaseFragment() {
...
         // If a student has submitted something, we can't let the teacher unpublish the assignment
-        if (!mAssignment.unpublishable) {
-            publishWrapper.setGone()
-            mIsPublished = true
-            return
-        }
         // Publish status
         publishSwitch.applyTheme()
         publishSwitch.isChecked = published
+        publishSwitch.isEnabled = mAssignment.unpublishable
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/teacher/src/main/java/com/instructure/teacher/fragments/EditAssignmentDetailsFragment.kt]: Fixed it, thanks! :)

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Review by hermannakos — APPROVED** [review]

**Pull request merged**